CorsixTH/Lua/epidemic.lua

```diff
@@ -411,6 +418,10 @@ later (@see applyOutcome) ]]
 function Epidemic:finishCoverUp()
   self.result_determined = true
 
+  if not self.inspector then
+    self:spawnInspector()
+  end
```

Comment:
I never played an epidemic, but an inspector always arrives, even if no infected uncured patients left the hospital?

If not, this looks like changed behavior

Reply:
> I never played an epidemic, but an inspector always arrives, even if no infected uncured patients left the hospital?
> 
> If not, this looks like changed behavior

Yeap, he should always arrive after CoverUp happened.
The only scenario when he won’t come is if you confess the epidemic and pay the fine straightaway, instead of doing CoverUp.
Currently, for some reason, the code was written so strangely, as if he could not always arrive after CoverUp.